ext/mysqlnd/mysqlnd_alloc.h

```diff
@@ -71,4 +71,16 @@ static inline MYSQLND_CSTRING mnd_str2c(const MYSQLND_STRING str)
 	return ret;
 }
 
+static inline void mysqlnd_set_message(MYSQLND_STRING *buf, const char *message, size_t len) {
```

Comment:
Why is this in mysqlnd_alloc now? This doesn't look allocation related.

Reply:
It's there because it didn't belong in connection.h either. What would be the best place to put it?

Reply:
At least with this name, I think connection.h is more appropriate. With a more generic name (`mysqlnd_set_string` or something?) this location may be fine.